README.md

```diff
@@ -9,35 +9,32 @@
 Please feel free to create [pull requests](https://github.com/keonkim/awesome-nlp/pulls).
 
 
-## Table of Contents
+## Contents
 
  - [Tutorials](#tutorials)
    - [Videos and Courses](#videos-and-online-courses)
  - [Deep Learning for NLP](#deep-learning-for-nlp)
- - [Packages](#packages)
-   - [Libraries](#libraries)
-     - [Node.js](#user-content-node-js)
-     - [Python](#user-content-python)
-     - [C++](#user-content-c++)
-     - [Java](#user-content-java)
-     - [Scala](#user-content-scala)
-     - [R](#user-content-R)
-     - [Clojure](#user-content-clojure)
-     - [Ruby](#user-content-ruby)
-     - [Rust](#user-content-rust)
-   - [Services](#services)
+ - [Libraries](#libraries)
+   - [Node.js](#user-content-node-js)
+   - [Python](#user-content-python)
+   - [C++](#user-content-c++)
+   - [Java](#user-content-java)
+   - [Scala](#user-content-scala)
+   - [R](#user-content-R)
+   - [Clojure](#user-content-clojure)
+   - [Ruby](#user-content-ruby)
+   - [Rust](#user-content-rust)
+ - [Services](#services)
  - [Techniques](#techniques)
    - [Word Vectors](#word-vectors)
    - [Thought Vectors](#thought-vectors)
    - [Machine Translation](#machine-translation)
-   - [General Natural Language Processing](#general-natural-language-processing)
    - [Single Exchange Dialogs](#single-exchange-dialogs)
    - [Memory and Attention Models](#memory-and-attention-models)
-   - [General Natural Language Processing](#general-natural-language-processing)
    - [Named Entity Recognition](#named-entity-recognition)
    - [Neural Network](#neural-network)
-   - [Supplementary Materials](#supplementary-materials)
- - [Blogs](#blogs)
+   - [Language Modeling](#language-modeling)
+   - [Question Answering and Knowledge Extraction](#question-answering-and-knowledge-extraction)
```

Comment:
Can we instead name it Natural Language Understanding?

Reply:
Yeah, we can split techniques and tasks as well if needed. I don't think it's needed right now.